sh.xaviers.Opentapes.yml

```diff
@@ -0,0 +1,45 @@
+app-id: sh.xaviers.Opentapes
+command: opentapes
+runtime: org.gnome.Platform
+runtime-version: '41'
+sdk: org.gnome.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.node16
+  - org.freedesktop.Sdk.Extension.rust-stable
+rename-icon: opentapes
+finish-args:
+  - --device=dri
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --socket=pulseaudio
+  - --share=network
+build-options:
+  append-path: /usr/lib/sdk/node16/bin:/usr/lib/sdk/rust-stable/bin
+  build-args:
+    - --share=network
+  env:
+    NPM_CONFIG_LOGLEVEL: info
+    CARGO_HOME: /run/build/Opentapes/cargo
+    RUST_BACKTRACE: '1'
+cleanup:
+  - /node_modules
+  - /dist
+  - /src-tauri/target
+modules:
+  - name: Opentapes
+    sources:
+      - type: archive
+        # For next release: https://codeberg.org/xaviers/Opentapes/archive/0.x.0.tar.gz
+        url: https://codeberg.org/xaviers/Opentapes/archive/main.tar.gz
+        sha256: 77a5143398b78cc86bc051b1c1372f7a38db8c2da92979375904c47db9ccd2bc
+    buildsystem: simple
+    build-commands:
+      - yarn install
+      - yarn build
+      - cargo install tauri-cli --version 1.0.0-beta.7
```

Comment:
this won't work, there is no network access at build time. flatpak-builder-tools should help you create the manifest for the dependencies for both cargo and yarn: https://github.com/flatpak/flatpak-builder-tools